playbook.yml

```diff
@@ -69,6 +69,10 @@ content:
     - url: https://github.com/OpenZeppelin/cairo-contracts
       branches: release-v*
       start_path: docs
+
+    - url: https://github.com/OpenZeppelin/nile
+      branches: [release-v*, docs-v*]
```

Comment:
This looks wrong, I believe it should just be `docs-v*` ? Otherwise there would be multiple branch matches for the same version.

Reply:
> This looks wrong, I believe it should just be `docs-v*` ?

It could be just `docs-v*`, but I wanted to keep `release-v*` because that's what we are using in cairo-contracts for releases, and the rationale of having an extra `docs-v*` is to allow "patches" to the doc-site outside releases (like fixing typos, format improvement, guides addition, etc...).

> Otherwise there would be multiple branch matches for the same version.

docs-v* would be created only outside releases, so versions shouldn't match (like docs-v0.12.0.1, being the last number the doc version outside a release). 

If I want to deploy the doc-site tomorrow after merging, I would use docs-v0.12.0.1 branch, because using release when we are not releasing nile feels wrong. And if I remove release-v*, I would be using a different convention for releases that the one we already have in cairo-contracts.

Agree a better format and a well-defined specification would be the right approach.